python/ray/train/v2/tests/test_failure_policy.py

```diff
@@ -19,15 +21,59 @@ def _worker_group_status_from_errors(errors):
     )
 
 
+def _worker_group_resize_status_from_errors():
```

Comment:
combine these 2 helpers

```suggestion
def _create_controller_error(retryable: bool = True):
```